### ISSUE TITLE
[App] Bug: Undefined behaviour for line plot in the table view

### ISSUE THREAD

**Issue by andreimargeloiu** [wandb/client#3149]
**Describe the bug**
I'm making a line plot in the table view to show the validation loss across multiple runs. However, I'm experiencing an unexpected behavior that I can't interpret, and I believe it's a bug.

![CleanShot 2022-01-17 at 10 21 19](https://user-images.githubusercontent.com/18227298/149751882-96e95d05-6159-4dac-964a-b41e66c25f6e.png)

A few things seem broken:
1. The legend. How is the number 4 selected? The legend is created based on "class_weight & valid_split" (see screenshot below). I tried different ways, and showing valid_split=4 seems random. The runs with valid_split are neither the longest (i.e., most training steps) nor have the highest performance.
2. The groups are unintuitive. How are the runs grouped? I wanted to plot all 10 runs (i.e., a group with the key "class_weight & valid_split" unique for all runs). Given the legend is broken, I don't understand the meaning for each plot.

**Screenshots**
Here are the settings in the plots:
![CleanShot 2022-01-17 at 10 23 50](https://user-images.githubusercontent.com/18227298/149752172-fd9ceea7-f736-42eb-9d58-ee0c58ee1134.png)

![CleanShot 2022-01-17 at 10 24 12](https://user-images.githubusercontent.com/18227298/149752199-e2e952d1-7373-43ca-bafd-0b85915abb5b.png)

![CleanShot 2022-01-17 at 10 24 21](https://user-images.githubusercontent.com/18227298/149752216-b94c22d5-bd43-409b-974f-32e02cc811ba.png)

![CleanShot 2022-01-17 at 10 24 29](https://user-images.githubusercontent.com/18227298/149752234-4a6029af-a8ea-4cb3-8973-7062799e31b5.png)

![CleanShot 2022-01-17 at 10 24 37](https://user-images.githubusercontent.com/18227298/149752278-e6df403f-a1aa-4583-861c-fe215a2821e2.png)

**Comment by andreimargeloiu** [comment]
The group's colors in the plots follow the "job type" groups. This seems to be the default grouping, but it's counterproductive in my use case.

How to disable this grouping?

**Comment by andreimargeloiu** [comment]
Update: When I add the same panel to a report, the panel behaves as expected and I can see all runs without being grouped. However, that's not the case from the table view.

![CleanShot 2022-01-17 at 11 14 55](https://user-images.githubusercontent.com/18227298/149760130-840d812a-d860-48cc-8e57-ebb4efb9536c.png)

**Comment by andreimargeloiu** [comment]
I've likely identified the issue. In the table view, the grouping runs toggle is activated (from the wandb backend), even though I explicitly deactivate it.

Expected behavior: when the toggle is deactivated, the runs shouldn't be grouped.

**The panel in the table view:**
![CleanShot 2022-01-17 at 11 17 22](https://user-images.githubusercontent.com/18227298/149760332-9ad1577e-1340-497a-baff-08bae34526cf.png)

**The same panel after being imported to reports:**
![CleanShot 2022-01-17 at 11 17 43](https://user-images.githubusercontent.com/18227298/149760357-135157da-ca53-4bec-849a-b7780e9bf397.png)

**Comment by andreimargeloiu** [comment]
Any update?

**Comment by aidanjdonohue** [comment]
Hi @andreimargeloiu , my apologies for the delay. Could you share a link to your wandb page where this is happening? It can be a private project.

**Comment by aidanjdonohue** [comment]
@andreimargeloiu , bumping this.

**Comment by aidanjdonohue** [comment]
Closing this out for now. Let us know if you'd like this reopened
